main.cpp

```diff
@@ -32,247 +77,347 @@ Create a branch named Part2
-    //5) Amount of Volume
+    
     int maxVolume = 1; //1-11
-//3 things it can do:
+
+    
+    
+    void osc2Mod(bool osc2ModActive = false);
     
-    //1) Modulate another Oscillator
-    bool osc2Mod = true;
-    //2) Generates a signal
     void signalGen(int frequency);
-    //3) Converts DC to AC
-    bool voltageConversion = true;
+    
+    void convertVoltage(bool voltageCovertActive = true);
 };
- 
+
+void Oscillator::osc2Mod(bool osc2ModActive)
+{
+    
+}
+
+void Oscillator::signalGen(int frequency)
+{
+
+}
+
+void Oscillator::convertVoltage(bool voltageConvertActive)
+{
+
+}
+
 
 
 //Thing 7) 
 struct LFO
 {
-    //1) Type of waveform
+    
     int waveType = 1; //1=sine 2=saw 3=square
-    //2) Rate of LFO
+    
     float LfoRate = 20.0f;
-    //3) Phase start position
+    
     int phaseStartPos = 1;  //1-100
-    //4) Sync with DAW Tempo
+    
     bool syncOn = false;
-    //5) Polarity
+    
     bool bipolarOn = true;
 
-//3 things it can do:
-    //1) Modulate Cutoff
+
+    
     void modulateCutoff(float lfoRate, bool syncOn = false);
-    //2) Change Pitch
+    
     void changePitch(int pitchChangeAmount);
-    //3) Adjust Frequency
+    
     void adjustFrequency(int freqHz);
 };
+void LFO::modulateCutoff(float lfoRate, bool syncOn)
+{
+
+}
+
+void LFO::changePitch(int pitchChangeAmount)
+{
+
+}
+void LFO::adjustFrequency(int freqHz)
+{
+
+}
+
+
 
 
 //Thing 8) 
 struct Display
 {
-    //1) Resolution
+    
     int screenRes = 1080;
-    //2) Pixel Size
+    
     int pixelSize = 2;
-    //3) Width
+    
     float displayWidth = 7.2f;
-    //4) Color
+    
     char displayColor = 'b'; //b = blue r = red
-    //5) Display Technology
+    
     std::string displayTechnology = "LED";
-//3 things it can do:
-    //1) Show settings
+
+    
     void displaySettings(char displayColor = 'r');
-    //2) Consume electric
-    void energyConsumption();
-    //3) Glow
+    
+    void energyConsumption(int wattsUsed);
+    
     void displayBritenessDim (bool powerSaver = true, int lumens = 100);
 };
  
+void Display::displaySettings(char displayColor)
+{
 
+}
+
+void Display::energyConsumption(int wattsUsed)
+{
 
+}
+
+void Display::displayBritenessDim (bool powerSaver, int lumens)
+{
+
+}
 //Thing 9) 
 struct FXSection
 {
-    //1) Amount of Reverb
+    
     int reverbAmount = 50;
-    //2) Amount of distortion
+    
     int distAmount = 1;
-    //3) Amount of Wet Signal (dry/wet)
+    
     int wetSignal = 50;
-    //4) Delay type
+    
     int delayType = 1; //1 = pingpong 2 = tape
-    //5) Reverb type
+    
     int reverbType = 1; //1 = plate 2 = spring
 
-//3 things it can do:
-    //1) Make the sound echo
+
+    
     void delayActive(bool delayOn = false); 
-    //2) Reverberate the sound
-    void reverbOutput(int reverbtype, bool reverbOn = true);
-    //3) Saturate the sound
-    void saturationActive(int distAmount, bool wetSignal());
+    
+    void reverbOutput(int reverbType, bool reverbOn = true);
```

Comment:
Your first parameter's name is the same as your member variable.  This is what the 'overshadows' warning is talking about.  You'll learn more about this in `Scope and Lifetime` later, but for now, change your function parameter's name.   If you see more 'overshadow' warnings, you'll need to fix them now that you know how to.
- [ ] revise